### ISSUE TITLE
Clean up unused imports removes imports used by attributes

### ISSUE THREAD

**Issue by jackolas-lock** [Tarik02/js-php-imports#3]
The VSCode extension that uses this package has been invaluable to me. I particularly like the feature that automatically removes unused packages. However, I've discovered it will remove imports used by attributes.

Steps to reproduce:

1. Add an attribute to a class or function which requires an import
2. Run the package with `php-imports.unused.enable` set to true
3. The import used by the attribute is removed.

Example: 
```
<?php

namespace App\Http\Controllers;

use App\Http\Resources\CareerResource;
use App\Models\Career;
use Illuminate\Http\Request;
use Vyuldashev\LaravelOpenApi\Attributes as OpenApi;

#[OpenApi\PathItem] // This attribute uses the OpenApi import above.
class CareerController extends Controller
{
    /**
     * Display a listing of the resource.
     */
     #[OpenApi\Operation] // This attribute also uses the OpenApi import above.
    public function index()
    {
        return Career->get();
    }
}
```

In this example, the `use Vyuldashev\LaravelOpenApi\Attributes as OpenApi;` is removed, even though it is necessary by those attributes.

**Comment by rasmus0201** [comment]
Any updates to this - would be a really nice thing to have! @tarik02

**Comment by tarik02** [comment]
@rasmus0201 no, I'm no longer working with PHP nor interested in having new features in this extension. Sorry.

**Comment by jonasclement** [comment]
Hi @tarik02,
I made a PR to fix this. If you are interested in merging it and updating this + the VS Code extension, please let me know. Otherwise I will publish my own forks of both, as my team uses the extension a lot :-)

**Comment by tarik02** [comment]
Thank you! If you wish to continue supporting this extension, I can add you permissions to this repository/move repository to your account.

**Comment by jonasclement** [comment]
If you have no interest in supporting this or the extension, I don't mind taking over. I can't/won't commit to full-scale support, but will gladly contribute what my team needs to it. :-)
